packages/bddeb

```diff
@@ -99,6 +101,36 @@ def write_debian_folder(root, templ_data, cloud_util_deps):
                              params={'build_depends': ','.join(requires)})
 
 
+def write_debian_folder_from_branch(root, templ_data, branch):
+    """Import a debian package directory from a branch."""
+    print("Importing debian/ from branch %s to %r" % (branch, root))
```

Comment:
```suggestion
    print("Importing debian/ from branch %s to %s" % (branch, root))
```

Reply:
The original `write_debian_folder()` I took this from uses `%r`, but `root` is a string, so +1 for the change.